### PR TITLE
Jackieriel added his info

### PR DESCRIPTION
**Pull request by Jackieriel**
FT-I am merging this file to yours

### CODE REVIEW

File: gitclass.html [hunk 1]
```diff
@@ -11,5 +11,8 @@
 <body>
    <h2>MY NAME IS AYOOLUWA</h2> 
    <h1>AM A SOFTWARE DEVELOPER</h1>
+   
+   <h3>My Name is jackieriel</h3>
```
Comment (lovisgod) [inline gitclass.html]: well jack, although you have done a great job. I want you to change the h3 to h2

**Review by lovisgod — COMMENTED** [review]
jack work on this

File: gitclass.html (hunk 1 above)
Comment (Jackieriel) [reply gitclass.html]: Ok am working on that

Comment (Jackieriel) [reply gitclass.html]: Am done with the changes

**Review by lovisgod — COMMENTED** [review]
Welldone jack. 
LGTM

**Pull request merged**